core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -203,6 +203,9 @@
     private Function<VirtualHost, Logger> accessLoggerMapper = host -> LoggerFactory.getLogger(
             defaultAccessLoggerName(host.hostnamePattern()));
 
+    private boolean includeServerHeader = true;
+    private boolean includeDateHeader = true;
```

Comment:
Which is better, `include`, `use` or `add`? I think `use` sounds more natural:

```java
Server.builder()
      .useServerHeader(false)
      .useDateHeader(false)
```

Reply:
Let's use `use` then. 😄

Reply:
I suggest new method name(if we enable `Server` and `Date` header by default).  🧐

```java
Server.builder()
      .disableServerHeader()
      .disableDateHeader()
```

Reply:
I think this is not a good idea. People will suddenly see the header after when they upgrade and be a bit surprised.

Reply:
Didn't know that the spec said sending `date` header is [mandatory](https://tools.ietf.org/html/rfc7231#section-7.1.1.2). Had a chat with @ikhoon and this is probably not a bad idea.
However, I think `disableDateHeader` can be a bit confusing because some could take that as not sending `date` header at all. So we should add `default` which is `disableDefaultDateHeader`. But it's too long, so I'm not sure which one is the best. 😄